privacy-policy.md

```diff
@@ -0,0 +1,37 @@
+# Discussify Privacy Policy
+##### First Posted: February 18, 2019
+
+The Discussify browser extension is owned by Protocol Labs Inc. and created by the IPFS Project. We know you care about how your personal data is used and we take your privacy seriously. __By using Discussify, you are accepting and agreeing to our collection and use of your information in the ways described below. At the moment, we only plan to use the information you provided to us for analytics to run and improve Discussify platform. We don’t plan to sell it to anyone else or use it for advertising. If that changes, we'll let you know by updating this policy.__
```

Comment:
Use `**` instead of `__` for consistency.

Reply:
Looks good. Can you add a line after headings? Also the heading levels seem weird because we start only with h4 onwards, perhaps decrease them a little bit?